OracleLinuxDevelopers/oraclelinux7/python/3.6-oracledb/Dockerfile

```diff
@@ -1,13 +1,18 @@
-# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
+# Copyright (c) 2020, 2023, Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 FROM oraclelinux:7-slim
 
-RUN yum -y install oraclelinux-developer-release-el7 oracle-instantclient-release-el7 && \
+RUN yum -y install oraclelinux-developer-release-el7 && \
     yum -y install python3 \
                    python3-libs \
                    python3-pip \
                    python3-setuptools \
-                   python36-cx_Oracle && \
+                   python3-oracledb && \
     rm -rf /var/cache/yum/*
 
+# Optionally install Oracle Instant Client to use python-oracledb Thick mode
+#RUN yum -y install oracle-instantclient-release-el7 && \
+#    yum -y install oracle-instantclient-basic && \
+#    rm -rf /var/cache/yum/*
```

Comment:
Can you please move these to the previous `RUN` directive so there's less chance folks have to download and rebuild the yum metadata twice? Also, "thick" doesn't need a capital letter. :)

Reply:
Our doc convention is to use initcaps for Thick and Thin since in some contexts it isn't apparent it is a name.